data/generalhw_scripts/get_sol_over_SSH.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
can you use `#!/bin/sh -e` by default to rely on POSIX sh and abort on errors. Use `#!/bin/bash -e` when you need actual bash features

Reply:
Fixed.